### PR TITLE
[12.0][ADD] hr_timesheet_report_not_valued

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
Adds a report that hides the time field.

### CODE REVIEW

File: hr_timesheet_report_not_valued/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        Adds a report that hides the time field.
+    """,
+    "version": "12.0.1.0.0",
+    "license": "LGPL-3",
+    "category": "After-Sales",
+    "author": "Solvos",
+    "website": "https://github.com/solvosci/slv-helpdesk",
+    "depends": [
+        "hr_timesheet",
+        "helpdesk_mgmt",
```
Comment (dalonsod) [inline hr_timesheet_report_not_valued/__manifest__.py]: The scope of this change shouldn't depend on this addon

File: hr_timesheet_report_not_valued/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Timesheet Report Not Valued",
+    "summary": """
+        Adds a report that hides the time field.
+    """,
+    "version": "12.0.1.0.0",
+    "license": "LGPL-3",
+    "category": "After-Sales",
```
Comment (dalonsod) [inline hr_timesheet_report_not_valued/__manifest__.py]: Better this:
```suggestion
    "category": "Human Resources",
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
See initial comments.

I propose a different approach: modifying current report without creating another one, and use parameters in order to apply changes, when report is called. I don't really know if it's possible, but I'd try it.

File: hr_timesheet_report_not_valued/i18n/es.po [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 2.4.3\n"
+"Last-Translator: \n"
+"Language: es_ES\n"
+
+#. module: hr_timesheet_report_not_valued
+#: model:ir.actions.report,name:hr_timesheet_report_not_valued.timesheet_not_valued_report
+msgid "Timesheet Entries Not Valued"
+msgstr "Detalle de los partes de horas no valoradas"
```
Comment (dalonsod) [inline hr_timesheet_report_not_valued/i18n/es.po]: Better this:
```suggestion
msgstr "Detalle de los partes de horas no valorado"
```

**Review by dalonsod — APPROVED** [review]
Tested standalone and interacting with other addons that also alters the same report (`helpdesk_mgmt_timesheet`, `helpdesk_mgmt_timesheet_ticket_name`,...).

See comment of `es.po` file.

I've looked for alternative on report xpath expressions, but I haven't found a better solution.

**Comment by dalonsod** [comment]
@lmiguens-solvos could you check?

**Review by lmiguens-solvos — APPROVED** [review]
LGTM

File: hr_timesheet_report_not_valued/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+{
+    "name": "Timesheet Report Not Valued",
+    "summary": """
+        Adds a report that hides the time field.
+    """,
+    "version": "12.0.1.0.0",
+    "license": "LGPL-3",
+    "category": "Human Resources",
+    "author": "Solvos",
+    "website": "https://github.com/solvosci/slv-helpdesk",
```
Comment (dalonsod) [inline hr_timesheet_report_not_valued/__manifest__.py]: Typo in manifest

```suggestion
    "website": "https://github.com/solvosci/slv-timesheet",
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
Missing `README` files and one comment on `manifest` file. When done, I'll merge it

**Pull request merged**